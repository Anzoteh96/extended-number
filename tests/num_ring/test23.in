241817513521925912808336051697245005900021309027095517491169608093229
820403945002019622221231889908678595392279675255013432109942481080082
-643937765994767092193424478420611849951170887859830377332931456135889
