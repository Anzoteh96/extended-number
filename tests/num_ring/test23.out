Enter the number base b:
Enter two numbers n and m:
b is 241817513521925912808336051697245005900021309027095517491169608093229
n mod b is 94951404436241883796223734816943577692215748173726879636433656800395
m mod b is 81514774571010646231583676671123167748893039221456175140577368143798
n + m is 176466179007252530027807411488066745441108787395183054777011024944193
n - m is 13436629865231237564640058145820409943322708952270704495856288656597
n * m is 188620902449216151826390503795920432702599852001454981434512840718209
n ** m is 188386139850931223568732274435059168010949391416079096602362034004367
